### PR TITLE
Whitelist signingKeyId param for add wallet

### PR DESCRIPTION
**Pull request by marktoda**
Whitelist signingKeyId param for add wallet

This commit whitelists the signingKeyId parameter for the wallet.add
function, which allows the user to specify a custom address to pay the
fees for a wallet creation transaction for certain coins.

This commit also fixes an npm audit issue for elliptic and dot-prop, and
fixes an XLM integration test issue

Ticket: BG-23036

### CODE REVIEW

**Comment by marktoda** [comment]
Corollary server-side PR: https://github.com/BitGo/bitgo-microservices/pull/4875

**Comment by marktoda** [comment]
Note this PR also fixes a couple critical vulnerabilities in transitive dependencies:
https://www.npmjs.com/advisories/1547
https://www.npmjs.com/advisories/1547

File: package.json [hunk 1]
```diff
@@ -43,5 +43,9 @@
     "kind-of": "^6.0.3",
     "minimist": "^1.2.2",
     "sanitize-html": "~1.20.0"
+  },
+  "resolutions": {
```
Comment (tylerlevine) [inline package.json:47]: can you merge this with the above section?

Comment (marktoda) [reply package.json:47]: lol, I didn't even see that - no problem

**Pull request merged**